### ISSUE TITLE
Docs/Workflow: Happy path for creating a new T2 Topology Application with Minimal Modules

### ISSUE THREAD

**Issue by tonyarkles** [zephyrproject-rtos/zephyr#60952]
**Is your enhancement proposal related to a problem? Please describe.**

I've been using Zephyr/West for about 6 months on a few different projects. I love it. I'd consider myself to be an intermediate user; I don't always know what the correct "working with the grain" approach is for some functionality, but I try very hard to follow Zephyr patterns. 

The problem I have is at the very start of a new project. I prefer the T2 Toplogy for my project structure (although am open to being told that I should consider a T1 or T3 topology instead), and generally struggle with bootstrapping such a project. My general approach involves creating a git repo on my laptop, stumbling my way through trying to make a manifest west.yml that will work, pushing that repo, and then trying to do a `west init` using it and hoping that I got all of the details right. After I do finally successfully create a workspace from my manifest, I delete my WIP repo and then work out of the west workspace that I managed to get going.

**Describe the solution you'd like**

- If there is an easier way to do this than what I'm doing, I'd like to make sure that the documentation for the workflow is easy to find. I haven't found it. I'd even be happy to *write and make a PR* for that documentation myself, but someone would need to tell me what the process I'm missing is.

- If there isn't an easier way to do this and people do generally like doing a T2 Topology project, maybe we could add some kind of template command that could use `west` to initialize the project with a valid template manifest *before* doing the `west update` that pulls down a bunch of BSPs that I don't need for my project.

Open to suggestions and happy to make the PR myself once the right approach is figured out.

**Comment by LysandrosAn** [comment]
Hi, this might come a bit late, but as a beginner user of Zephyr, I fully agree that the entire workflow for setting up projects is not very clear. Especially because not anyone pushes sandbox projects up to git. Often people prefer to copy-paste project folders for trying out code
